cmd/podman/machine/inspect.go

```diff
@@ -90,8 +90,7 @@ func inspect(cmd *cobra.Command, args []string) error {
 			SSHConfig:          mc.SSH,
 			State:              state,
 			UserModeNetworking: false,
-			// TODO I think this should be the HostUser
-			Rootful: mc.HostUser.Rootful,
+			HostUser:           mc.HostUser,
```

Comment:
Is this accurate? I just heard in another PR that this is always the rootless user name - if the machine is rootful, this is not "root"

Reply:
Do you have a link to the PR?

Reply:
Hmm. Maybe we have the a `RootlessUsername` or a `MachineUsername` field instead? In any case, HostUser mostly doesn't look that useful in inspect, other than rootful. It would be confusing if it was called HostUser and was actually just the rootless usernaname
```
type HostUser struct {
	// Whether this machine should run in a rootful or rootless manner
	Rootful bool
	// UID is the numerical id of the user that called machine
	UID int
	// Whether one of these fields has changed and actions should be taken
	Modified bool `json:"HostUserModified"`
}
```

Reply:
If it's information isn't really that useful other than `Rootful` I'm fine with reverting the changes

Reply:
In our mtg today I think we decided on keeping `Rootful` and also adding in a field, `MachineUser`, that displays the username inside the machine.

Reply:
@ashley-cui @baude the username inside the machine ("core") is already stored at `InspectInfo.SSHConfig.RemoteUsername`. I don't think we need another field for it, unless I'm not understanding this properly

Reply:
We can keep just one, but the important bit is that we are keeping Rootful

Reply:
Changes reverted. Rootful still remains the same